test/e2e/example.spec.js

```diff
@@ -0,0 +1,19 @@
+// @ts-check
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
+
+  // Expect a title "to contain" a substring.
```

Comment:
```suggestion
  // Expect a title "to contain" a substring.
```

We usually don't write very general comments. Please remove these kind of comments below also.

Reply:
You can remove the whole file. It's just an example test file.